aks/yaml/stats-deploy.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: statistics-service-deployment
+  labels:
+    app: stats-deploy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: stats-deploy
+  template:
+    metadata:
+      labels:
+        app: stats-deploy
+    spec:
+      containers:
+        - name: stats
+          # docker.io/marcinziolkowski/stats-service:v1.0.1
```

Comment:
Is this comment necessary?

Reply:
i fixed this on some other branch - some time ago docker defaulted to a different image registry, this is no longer needed